adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala

```diff
@@ -98,23 +103,73 @@ object NucleotideContigFragmentRDD extends Serializable {
   def apply(rdd: RDD[NucleotideContigFragment],
             sequences: SequenceDictionary): NucleotideContigFragmentRDD = {
 
-    NucleotideContigFragmentRDD(rdd, sequences, None)
+    RDDBoundNucleotideContigFragmentRDD(rdd, sequences, None)
+  }
+}
+
+case class ParquetUnboundNucleotideContigFragmentRDD private[rdd] (
```

Comment:
Yeah, all the constructors should be `private[rdd]`. I'll go back through and get the ones that I've missed.